src/nvapi_gpu.cpp

```diff
@@ -289,6 +293,78 @@ extern "C" {
         return Ok(n);
     }
 
+    // Function to check for WAR to DLSS Bug 3634851 if an affected DLSS DLL is
+    // detected this function will return true; false will be returned for all
+    // non-affected DLLs.
+    static bool isDLSSBug3634851WARNeeded(void* pReturnAddress) {
```

Comment:
See below about the suggestion to move to `env`.
The bug ID is unfortunately private, so I would prefer a more descriptive name. That said please leave the ID (and/or a private link to it) in source code comments.